content/QCM Physics/01_Crystals/Crystals.md

```diff
@@ -32,7 +35,7 @@ How to draw it in 2D: ^69cf07
 1) Find the point for which you want to draw the cell.
 2) Draw lines that connect it to all its nearest neighbours.
 3) For each of these lines draw a perpendicular line(*) that passes through its mid point.
-4) The area delimited by the lines (*) is the Wigner-Seitz cell.
```

Comment:
The (*) is there to indicate the lines in the previous step. (P.S. I don't know what I'm doing with pull requests)